src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationMiddleware.cs

```diff
@@ -46,9 +45,14 @@ public class OpenIdConnectAuthenticationMiddleware : AuthenticationMiddleware<Op
         {
             _logger = loggerFactory.CreateLogger<OpenIdConnectAuthenticationMiddleware>();
 
+            if (string.IsNullOrEmpty(Options.SignInScheme))
+            {
+                Options.SignInScheme = externalOptions.Options.SignInScheme;
+            }
```

Comment:
So, we are talking about SignInScheme that lives on ExternalAuthenticationOptions correct?  This should be optional always, since its only useful when forwarding a "passive" (new terminology is Automatic = false), auth middleware's ticket to something automatic like bearer/cookie for persistence.  We probably should rename this to convey the usage.  Perhaps something like SignInForwardingOptions && SignInScheme maybe?  @lodejard what do you think?